### ISSUE TITLE
Does not send aftertouch in "Live" mode

### ISSUE THREAD

**Issue by vincenzoml** [git-moss/LaunchpadPro4Bitwig#4]
In "live" mode, the launchpad pro does not send aftertouch to bitwig (I'm using linux and bw 1.3.13). In "note" mode, aftertouch is correctly sent (tested with polysynth).

**Comment by git-moss** [comment]
What do you mean with "live mode"?

**Comment by vincenzoml** [comment]
Hi, the issue is solved it seems by fiddling with pressure sensitivity using the "setup" button; so feel free to close it. However there is something odd about the script right now. The "live" mode I refer to is the mode that the device enters if you press "setup" and the top-left-most key. That should be entered automatically when bitwig connects to the device, but it doesn't. I have to select it by hand. I think the very first day I used the device this was not the case and I associated it to the real problem I have: the volume, pan, and sends button *don't* work at all. They switch the device to "session" view and a message appears on screen, but the device keeps sending notes. Is this related to usage of the three midi ports of the launchapd? I am using the same midi port for send and receive by bitwig, maybe this is wrong? But if I do otherwise the lights don't turn on.
